t/t0019-tbon-config.t

```diff
@@ -79,6 +79,15 @@ test_expect_success 'tbon.interface-hint=hostname works' '
 	flux start -o,-Stbon.interface-hint=hostname,-Stbon.prefertcp=1 \
 		${ARGS} -s2 true
 '
+test_expect_success 'tcp.interface-hint defaults to default-router' '
+	flux start ${ARGS} flux getattr tbon.interface-hint >defhint.out &&
+	grep default-route defhint.out
+'
+test_expect_success 'tcp.interface-hint setting is child default' '
+	flux start -o,-Stbon.interface-hint=hostname \
+	    flux alloc -N1 flux getattr tbon.interface-hint >childhint.out &&
+	grep hostname childhint.out
+'
```

Comment:
Should a test be added that ensures setting the attribute in the subinstance uses the set value and not the inherited one?